app/templates/events/view/speakers.hbs

```diff
@@ -16,20 +16,18 @@
         {{/link-to}}
       {{/tabbed-navigation}}
     </div>
-    <div class="eight wide column">
-      <button class="ui blue button right floated">{{t 'Add Speaker'}}</button>
-      <button class="ui blue button right floated">{{t 'Export as CSV'}}</button>
-      {{#ui-dropdown class='labeled icon button right floated'}}
-        <i class="dropdown icon"></i>
-        <div class="default text">{{t 'No. of entries'}}</div>
-        <div class="menu">
-          <div class="item">10</div>
-          <div class="item">25</div>
-          <div class="item">50</div>
-          <div class="item">100</div>
-        </div>
-      {{/ui-dropdown}}
-    </div>
+    {{#if device.isMobile}}
```

Comment:
@harshitagupta30 The solutions works perfectly. However there is a lot of redundant code. We can get the same outcome using the following code:
```
 <div class="ten wide {{if device.isMobile 'center aligned'}} column">
   <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Export as CSV'}}</button>
   {{#if device.isMobile}}
    <div class="ui fitted divider"></div>
   {{/if}}
   <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Add Speaker'}}</button>
 </div>
```